bootstrap-salt.sh

```diff
@@ -1338,7 +1350,11 @@ __check_dpkg_architecture() {
             ;;
     esac
 
-    if [ "${error_msg}" != "" ]; then
+    if [ "${warn_msg}" != "" ]; then
+        # AArch64: Do not fail at this point, but warn the user about experimental support
+        # See https://github.com/saltstack/salt-bootstrap/issues/1240
+        echowarn "${warn_msg}"
+    elif [ "${error_msg}" != "" ]; then
```

Comment:
Do we really want to display either `{warn_msg}` or `${error_msg}`? Maybe better to output them both when they are defined...